### ISSUE TITLE
Tracks found: 0(python version:3.7.7, torch:1.3.0, torchvision: 0.4.1)

### ISSUE THREAD

**Issue by chenrxi** [phil-bergmann/tracking_wo_bnw#117]
WARNING - test_tracktor - No observers have been added to this run
INFO - test_tracktor - Running command 'main'
INFO - test_tracktor - Started
Configuration (modified, added, typechanged, doc):
  seed = 453695
  reid:
    db_train = 'train'
    db_val = False
    desription = 'all'
    lr_scheduler = True
    max_epochs = 69
    module_name = 'reid'
    name = 'res50-mot17-batch_hard'
    seed = 12345
    cnn:
      output_dim = 128
    dataloader:
      K = 4
      P = 18
      crop_H = 256
      crop_W = 128
      max_per_person = 1000
      normalize_mean = [0.485, 0.456, 0.406]
      normalize_std = [0.229, 0.224, 0.225]
      transform = 'random'
      vis_threshold = 0.3
    model_args:
      loss = 'batch_hard'
      margin = 0.2
      prec_at_k = 3
    solver:
      optim = 'Adam'
      optim_args:
        lr = 0.0003
        weight_decay = 0.0
  tracktor:
    dataset = 'mot17_train_FRCNN17'
    desription = None
    frame_split = [0.0, 1.0]
    interpolate = False
    module_name = 'MOT17'
    name = 'Tracktor++'
    network = 'fpn'
    obj_detect_model = 'output/faster_rcnn_fpn_training_mot_17/model_epoch_27.model'
    reid_config = 'output/tracktor/reid/res50-mot17-batch_hard/sacred_config.yaml'
    reid_weights = 'output/tracktor/reid/res50-mot17-batch_hard/ResNet_iter_25245.pth'
    seed = 12345
    write_images = False
    tracker:
      detection_nms_thresh = 0.3
      detection_person_thresh = 0.5
      do_align = True
      do_reid = True
      inactive_patience = 10
      max_features_num = 10
      number_of_iterations = 100
      public_detections = True
      regression_nms_thresh = 0.6
      regression_person_thresh = 0.5
      reid_iou_threshold = 0.2
      reid_sim_threshold = 2.0
      termination_eps = 1e-05
      warp_mode = 'cv2.MOTION_EUCLIDEAN'
      motion_model:
        center_only = True
        enabled = False
        n_steps = 1
INFO - main - Initializing object detector.
INFO - main - Tracking: MOT17-02-FRCNN
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 600/600 [01:15<00:00,  7.97it/s]
INFO - main - Tracks found: 0
INFO - main - Runtime for MOT17-02-FRCNN: 75.33 s.
INFO - main - Writing predictions to: /home/chenruoxi/tracking_wo_bnw/output/tracktor/MOT17/Tracktor++
INFO - main - Tracking: MOT17-04-FRCNN

What can i do?

**Comment by timmeinhardt** [comment]
Hello, please double check if the dataset is in the correct directory and if you completed all other steps of the README.md.

**Comment by MGudik** [comment]
I found that:
public_detections = True
was searching for external detections which could not be found in my case, then it found 0 tracks.

Set it to False, solved my issue.

**Comment by timmeinhardt** [comment]
I suspect a dataset issue in both cases. The `MOT17` dataset folder includes public detection sets.

**Comment by MGudik** [comment]
Hmm, true. I thought it was a custom dataset, I was a bit too quick in my reply :)
